server/image_pull.go

```diff
@@ -96,7 +96,7 @@ func (s *Server) PullImage(ctx context.Context, req *types.PullImageRequest) (*t
 
 	if pullOp.err != nil {
 		if errors.Is(pullOp.err, syscall.ECONNREFUSED) {
-			return nil, k8simages.ErrRegistryUnavailable
+			return nil, crierrors.ErrRegistryUnavailable
```

Comment:
not a blocker, but the PR would be easy to read if you push this change in a separate commit.

Reply:
+1 I agree

Reply:
Done, split-up the commits.